### PR TITLE
Refactor scales in TimelineTicks

### PR DESCRIPTION
**Pull request by karupayun**
After a discussion with Carsten, I'm simplifying scales' logic to be
easier to understand.

Also I'm erasing "15s" and "15min" because they break a desired property.

### CODE REVIEW

File: src/OrbitGl/TimelineTicks.cpp [hunk 1]
```diff
@@ -8,38 +8,39 @@
 namespace orbit_gl {
 
 static std::set<uint64_t> GenerateScales() {
-  std::set<uint64_t> scales;
+  // 10^x for each unit and middle points.
```
Comment (beckerhe) [inline src/OrbitGl/TimelineTicks.cpp:11]: Nit: I don't get the middle points part of the comment. Aren't the middle points NOT added to this set?

**Review by beckerhe — APPROVED** [review]
LGTM

File: src/OrbitGl/TimelineTicks.cpp (hunk 1 above)
Comment (karupayun) [reply src/OrbitGl/TimelineTicks.cpp:11]: This comment was intended to be part of the entire function. Shall I put it outside the body?

Comment (beckerhe) [reply src/OrbitGl/TimelineTicks.cpp:11]: Ah makes sense. Yes in that case please above the function!

**Pull request merged**